akka-docs/src/test/java/jdocs/actor/ActorDocTest.java

```diff
@@ -791,7 +791,7 @@ public void useKill() {
         victim.tell(akka.actor.Kill.getInstance(), ActorRef.noSender());
         
         // expecting the actor to indeed terminate:
-        expectTerminated(Duration.create(3, TimeUnit.SECONDS), victim);
+        expectTerminated(scala.concurrent.duration.Duration.create(3, TimeUnit.SECONDS), victim);
```

Comment:
This one we should add to the Java testkit with Java duration as well, but we can do that separately from this PR